openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -40,11 +40,13 @@ def get_ocio_config_path(profile_folder):
     Returns:
         str: Path to vendorized config file.
     """
+
     return os.path.join(
         os.environ["OPENPYPE_ROOT"],
         "vendor",
-        "configs",
-        "OpenColorIO-Configs",
+        "bin",
+        "ocioconfig"
```

Comment:
This is missing a comma I believe
from `"ocioconfig"` to `"ocioconfig",`

Reply:
```suggestion
        "ocioconfig",
```